tests/switchless/host/switchless_args.h

```diff
@@ -0,0 +1,68 @@
+/*
+ *  This file is auto generated by oeedger8r. DO NOT EDIT.
+ */
+#ifndef EDGER8R_SWITCHLESS_ARGS_H
+#define EDGER8R_SWITCHLESS_ARGS_H
+
+#include <stdint.h>
+#include <stdlib.h> /* for wchar_t */
+
+/* #include <errno.h> - Errno propagation not enabled so not included. */
+
+#include <openenclave/bits/result.h>
```

Comment:
Can you create an issue to remove these auto-generated files in test

Reply:
The auto-generated files are now removed.